JustSaying.IntegrationTests/JustSaying.IntegrationTests.csproj

```diff
@@ -60,6 +60,12 @@
     <Reference Include="Shouldly">
       <HintPath>..\packages\Shouldly.1.1.1.1\lib\35\Shouldly.dll</HintPath>
     </Reference>
+    <Reference Include="StructureMap">

```

Comment:
This dependency is just for integration test purposes. JustSaying library has no dependency on any IoC container.